mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -3456,6 +3456,30 @@ func TestAccContainerCluster_withEnableKubernetesAlpha(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_withEnableKubernetesBetaAPIs(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+	npName := fmt.Sprintf("tf-test-np-%s", acctest.RandString(t, 10))
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
If possible, we should add a second step here that adds a second enabled API to exercise the Update code. If that isn't possible because there is only one beta API, we should add a new test that creates a cluster without any enabled APIs and then enables this beta API in a separate test step.

Reply:
Unfortunately, there is only one beta API available at this time. Added a new test case to create a GKE cluster and enable the beta API.

Reply:
Thanks @toVersus  yeah we don't have any other APIs for enabling